marlowe-playground-client/src/Blockly/Events.js

```diff
@@ -0,0 +1,8 @@
+exports._unsafeReadBlocklyEventType = function (nothing, just, name, value) {
+    var proto = Object.getPrototypeOf(value);
+    if ('type' in proto && proto.type === name) {
+      return just(value);
+    } else {
+      return nothing;
+    }
+}
```

Comment:
does this need to be called unsafe? If you add a few null checks in then I would say it's safe?

Reply:
I'm happy to remove the unsafe word as I think is safe enough, but I based from the [MessageEvent](https://github.com/purescript-web/purescript-web-socket/blob/v2.0.0/src/Web/Socket/Event/MessageEvent.purs#L11-L11) implementation that uses [unsafeReadProtoTagged](https://pursuit.purescript.org/packages/purescript-web-events/2.0.1/docs/Web.Internal.FFI#v:unsafeReadProtoTagged). If they consider that function is unsafe, I thought the one I made is unsafe too 🤷‍♂️

Reply:
done